100%|██████████| 11/11 [00:00<00:00, 41.32it/s]
100%|██████████| 11/11 [00:00<00:00, 41.70it/s]
[2024-01-31 17:01:31,667][root][INFO] - using 1 gpus
the PU data is ['KA04', 'KA15', 'KA16', 'KA22', 'KA30', 'KB23', 'KB24', 'KB27', 'KI14', 'KI17', 'KI21']
the PU data is ['KA04', 'KA15', 'KA16', 'KA22', 'KA30', 'KB23', 'KB24', 'KB27', 'KI14', 'KI17', 'KI21']
--------PU Dataset:(1, 0) TL Task---------
the input kind is fft,the model kind is ResNet18
the opt infor is {'name': 'adamw', 'lr_src': 0.001, 'lr_tar': 0.001, 'lr_scheduler': 'designed', 'weight_decay_src': 0.0001, 'weight_decay_tar': 0.0001}
the data infor is {'data_name': 'PU', 'data_path': 'D:/P_new/PU_dataset', 'TL_list': [0, 1, 2], 'input_kind': 'fft', 'norm_kind': 'mean-std', 'TL_Task': [1, 0]}
[2024-01-31 17:01:35,024][urllib3.connectionpool][WARNING] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))': /api/4504800232407040/envelope/
Task: [1, 0], Iter:1/50; Accuracy_S/T = 89.31%/ 80.40%; Loss= 1.12
Task: [1, 0], Iter:2/50; Accuracy_S/T = 99.93%/ 92.20%; Loss= 0.63
[2024-01-31 17:01:45,408][urllib3.connectionpool][WARNING] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))': /api/4504800232407040/envelope/
Task: [1, 0], Iter:3/50; Accuracy_S/T = 96.15%/ 91.87%; Loss= 0.61
[2024-01-31 17:01:55,790][urllib3.connectionpool][WARNING] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))': /api/4504800232407040/envelope/
Task: [1, 0], Iter:4/50; Accuracy_S/T = 99.96%/ 94.74%; Loss= 0.59
Task: [1, 0], Iter:5/50; Accuracy_S/T = 99.93%/ 95.10%; Loss= 0.58
Task: [1, 0], Iter:6/50; Accuracy_S/T = 100.00%/ 93.28%; Loss= 0.57
Task: [1, 0], Iter:7/50; Accuracy_S/T = 100.00%/ 95.21%; Loss= 0.57
[2024-01-31 17:02:16,631][urllib3.connectionpool][WARNING] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))': /api/4504800232407040/envelope/
Task: [1, 0], Iter:8/50; Accuracy_S/T = 99.67%/ 89.66%; Loss= 0.57
Task: [1, 0], Iter:9/50; Accuracy_S/T = 100.00%/ 93.39%; Loss= 0.56
[2024-01-31 17:02:27,033][urllib3.connectionpool][WARNING] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))': /api/4504800232407040/envelope/
Task: [1, 0], Iter:10/50; Accuracy_S/T = 100.00%/ 92.78%; Loss= 0.56
[2024-01-31 17:02:37,425][urllib3.connectionpool][WARNING] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)'))': /api/4504800232407040/envelope/
Task: [1, 0], Iter:11/50; Accuracy_S/T = 100.00%/ 94.56%; Loss= 0.56
Task: [1, 0], Iter:12/50; Accuracy_S/T = 100.00%/ 94.30%; Loss= 0.56
Task: [1, 0], Iter:13/50; Accuracy_S/T = 100.00%/ 90.34%; Loss= 0.56
Task: [1, 0], Iter:14/50; Accuracy_S/T = 100.00%/ 94.30%; Loss= 0.56
Task: [1, 0], Iter:15/50; Accuracy_S/T = 100.00%/ 94.63%; Loss= 0.56
Task: [1, 0], Iter:16/50; Accuracy_S/T = 100.00%/ 94.70%; Loss= 0.56
Task: [1, 0], Iter:17/50; Accuracy_S/T = 100.00%/ 94.70%; Loss= 0.56
Task: [1, 0], Iter:18/50; Accuracy_S/T = 100.00%/ 94.37%; Loss= 0.55
Task: [1, 0], Iter:19/50; Accuracy_S/T = 100.00%/ 94.48%; Loss= 0.56
Task: [1, 0], Iter:20/50; Accuracy_S/T = 100.00%/ 94.37%; Loss= 0.56
Task: [1, 0], Iter:21/50; Accuracy_S/T = 100.00%/ 94.48%; Loss= 0.55
Task: [1, 0], Iter:22/50; Accuracy_S/T = 100.00%/ 93.54%; Loss= 0.56
Task: [1, 0], Iter:23/50; Accuracy_S/T = 100.00%/ 93.61%; Loss= 0.55
Task: [1, 0], Iter:24/50; Accuracy_S/T = 100.00%/ 94.01%; Loss= 0.55
Task: [1, 0], Iter:25/50; Accuracy_S/T = 100.00%/ 94.23%; Loss= 0.55
Task: [1, 0], Iter:26/50; Accuracy_S/T = 100.00%/ 93.97%; Loss= 0.55
Task: [1, 0], Iter:27/50; Accuracy_S/T = 100.00%/ 93.65%; Loss= 0.55
Task: [1, 0], Iter:28/50; Accuracy_S/T = 100.00%/ 93.54%; Loss= 0.55
Task: [1, 0], Iter:29/50; Accuracy_S/T = 100.00%/ 94.23%; Loss= 0.55
Task: [1, 0], Iter:30/50; Accuracy_S/T = 100.00%/ 94.16%; Loss= 0.55
Task: [1, 0], Iter:31/50; Accuracy_S/T = 100.00%/ 93.50%; Loss= 0.55
Task: [1, 0], Iter:32/50; Accuracy_S/T = 100.00%/ 94.88%; Loss= 0.55
Task: [1, 0], Iter:33/50; Accuracy_S/T = 100.00%/ 93.76%; Loss= 0.55
Task: [1, 0], Iter:34/50; Accuracy_S/T = 100.00%/ 93.14%; Loss= 0.55
Task: [1, 0], Iter:35/50; Accuracy_S/T = 100.00%/ 94.26%; Loss= 0.55
Task: [1, 0], Iter:36/50; Accuracy_S/T = 100.00%/ 94.45%; Loss= 0.55
Task: [1, 0], Iter:37/50; Accuracy_S/T = 100.00%/ 93.54%; Loss= 0.55
Task: [1, 0], Iter:38/50; Accuracy_S/T = 100.00%/ 94.12%; Loss= 0.55
Task: [1, 0], Iter:39/50; Accuracy_S/T = 100.00%/ 94.01%; Loss= 0.55
Task: [1, 0], Iter:40/50; Accuracy_S/T = 100.00%/ 94.30%; Loss= 0.55
Task: [1, 0], Iter:41/50; Accuracy_S/T = 100.00%/ 94.16%; Loss= 0.55
Task: [1, 0], Iter:42/50; Accuracy_S/T = 100.00%/ 94.56%; Loss= 0.55
Task: [1, 0], Iter:43/50; Accuracy_S/T = 100.00%/ 94.19%; Loss= 0.55
Task: [1, 0], Iter:44/50; Accuracy_S/T = 100.00%/ 94.08%; Loss= 0.55
Task: [1, 0], Iter:45/50; Accuracy_S/T = 100.00%/ 94.77%; Loss= 0.55
Task: [1, 0], Iter:46/50; Accuracy_S/T = 100.00%/ 92.49%; Loss= 0.55
Task: [1, 0], Iter:47/50; Accuracy_S/T = 100.00%/ 92.81%; Loss= 0.55
Task: [1, 0], Iter:48/50; Accuracy_S/T = 100.00%/ 93.28%; Loss= 0.55
Task: [1, 0], Iter:49/50; Accuracy_S/T = 100.00%/ 93.50%; Loss= 0.55
Task: [1, 0], Iter:50/50; Accuracy_S/T = 100.00%/ 93.68%; Loss= 0.55